iohub.util.exception_tools.print2err
====================================

.. currentmodule:: iohub.util.exception_tools

.. autofunction:: print2err